apps/gdalbuildvrt_bin.cpp

```diff
@@ -56,6 +56,7 @@ static void Usage(bool bIsError, const char *pszErrorMsg)
         "                    [-srcnodata \"<value>[ <value>]...\"] [-vrtnodata "
         "\"<value>[ <value>]...\"\n"
         "                    [-ignore_srcmaskband]\n"
+        "                    [-nodata_if_mask_less_or_equal <threshold>]\n"
```

Comment:
Small nit... `-nodata_max_mask_threshold` reads a bit cleaner to me, but it is kind of weird to have options right next to each other where some are snake case and others are arbitrarily smashed together (👀 at `srcmaskband`).

Reply:
yeah option naming is the hardest part... Your suggestion looks nicer indeed.

Reply:
option renamed